src/sql/workbench/services/objectExplorer/common/objectExplorerService.ts

```diff
@@ -124,6 +124,8 @@ export interface TopLevelChildrenPath {
 	providerObject: azdata.ObjectExplorerNodeProvider | azdata.ObjectExplorerProvider;
 }
 
+const errSessionCreateFailed = nls.localize('OeSessionFailedError', "Failed to create Object Explorer session");
```

Comment:
Oe looks weird to me. But it’s fine.

Reply:
I mean I'm using what already was there, just moving it.